src/js/components/Select/doc.js

```diff
@@ -77,6 +77,9 @@ export const doc = Select => {
     focusIndicator: PropTypes.bool.description(
       "Whether when 'plain' it should receive a focus outline.",
     ),
+    icon: PropTypes.oneOfType([PropTypes.bool, PropTypes.node]).description(
+      'The icon to be used when rendering the select. You can use false to not render an icon at all.',
+    ),
```

Comment:
I think it is missing `.defaultValue(true)`

Reply:
actually the default value is undefined.

Reply:
We need to communicate that the icon is shown by default

Reply:
is that really needed? we dont communicate in other scenarios. take the `focusIndicator` right there... we don't say "we show the focus indicator by default".

Reply:
how about this for the description?

> A custom icon to be used when rendering the select. You can use false to not render an icon at all.

Reply:
That should work, thanks